src/applications/vaos/project-cheetah/redux/actions.js

```diff
@@ -203,6 +222,96 @@ export function updateFacilitySortMethod(sortMethod, uiSchema) {
   };
 }
 
+export function getAppointmentSlots(startDate, endDate, forceFetch = false) {
+  return async (dispatch, getState) => {
+    const state = getState();
+    const useVSP = selectFeatureVSPAppointmentNew(state);
+    const siteId = getSiteIdFromFakeFHIRId(
+      selectProjectCheetahFormData(state).vaFacility,
+    );
+    const newBooking = selectProjectCheetahNewBooking(state);
+    const { data } = newBooking;
+
+    data.typeOfCareId = '301';
+    data.clinicId = 'var983_455';
```

Comment:
You can delete this line now, the clinic id should already be set by the clinic page.